docs/lti-integration.md

```diff
@@ -0,0 +1,141 @@
+# LTI 1.3 Integration
+
+We are working to fully support LTI 1.3 as an LTI Tool. Currently, we are beta
+testing with a small group of Canvas users, and plan to release phases of LTI 1.3
+features in 2024.
+
+Below are instructions on how to register Code.org as an LTI Tool in Canvas.
+NOTE: this pilot is currently supported for a closed group, and is not get
+generally available.
+
+### References
+
+- [Create a new LTI Developer Key][install-canvas]
+- [Configure the LTI app][configure-canvas]
+
+[install-canvas]: https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140
+[configure-canvas]: https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-external-app-for-an-account-using-a-client/ta-p/202
+
+## Installing Code.org in Canvas
+
+See [link here][install-canvas] for detailed step by step instructions for
+configuring a developer key for an LTI Tool, i.e. installing an LTI Tool in
+Canavs.
+
+In the `Select Configuration Method` step, there are two options, detailed below.
+For both options, you must first configure your `Key Settings`:
+
+1. Key Name: `<your-key-name>`
+1. Owner Email: `<admin-owner-email>`
+1. Redirect URIs: `https://studio.code.org` (NOTE: This will prefill automatically if you chose the JSON method)
+1. Notes: `Optional: Any notes about the LTI key, such as the reason it was created.`
+
+### Enter Manual Entry Details
+
+This is the first option in the configuration steps. The easier method is the
+JSON method detailed below, however here are instructions for manually entering
+these details.
```

Comment:
Maybe we should put the recommended/easier option first in instructions even though it's not the first presented option?

Reply:
+1, simple first imo

Reply:
The reason for the ordering is to match the Canvas directions that are linked here. I agree, I'd prefer to put simple first, just thought it might be confusing to jump the ordering of the instructions they are walking through. Thoughts?

Reply:
Ok, let me rework this a bit. What I'm gathering from your above outline is break up the flow into larger buckets (Steps) and call out Code.org specifics in those.

Couple clarifications/answers:

> can we explicitly call out the steps we expect the admins to take via the external links?

We expect them to follow the Canvas instructions completely. There is not a step they would skip. I'll refactor these instructions to help call out specifics where they will need to enter Code.org values.

> Step 4) Generate a deployment ID (maybe we can pull in screenshots for this since no anchor links)

This requires they follow the second set of linked instructions, which contains multiple steps and multiple screen shots to generate the deployment id. Including a single screenshot here won't be much help, as it's a more than a single step.